src/nanosvg.h

```diff
@@ -2888,6 +2882,7 @@ NSVGimage* nsvgParseFromFile(const char* filename, const char* units, float dpi)
 	if (!fp) goto error;
 	fseek(fp, 0, SEEK_END);
 	size = ftell(fp);
+	if (size <= 0) goto error;
```

Comment:
something here is probably wrong, size should not be a size_t but a long.  size_t will never be < 0 but long can be, and will be -1 if ftell fails.

Reply:
Of course the type is wrong. Should have fixed this too.